{
	"identifier": "CVE-2018-1112",
	"description": "glusterfs server before versions 3.10.12, 4.0.2 is vulnerable when using 'auth.allow' option which allows any unauthenticated gluster client to connect from any network to mount gluster storage volumes. NOTE: this vulnerability exists because of a CVE-2018-1088 regression.",
	"type": "network",
	"severity": "high",
	"state": "published",
	"datetime": "2018-04-25T10:29:00Z",
	"is_edited": false,
	"hardware": [],
	"software": [
		{
			"name": "glusterfs",
			"version": "\u003c 9.2-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "glusterfs",
			"version": "\u003c 5.5-3",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		},
		{
			"name": "glusterfs",
			"version": "\u003c 10.3-5",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-sid"
		},
		{
			"name": "glusterfs",
			"version": "\u003c 10.3-5",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-trixie"
		},
		{
			"name": "glusterfs",
			"version": "\u003c 10.3-5",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bookworm"
		},
		{
			"name": "glusterfs",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "glusterfs",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-xenial"
		},
		{
			"name": "glusterfs",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-artful"
		},
		{
			"name": "glusterfs",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-bionic"
		}
	],
	"products": [
		{
			"name": "glusterfs",
			"version": "\u003c 3.10.12",
			"architecture": "any",
			"vendor": "gluster",
			"type": "software",
			"is_edited": false,
			"aliases": []
		},
		{
			"name": "glusterfs",
			"version": "4.0.2",
			"architecture": "any",
			"vendor": "gluster",
			"type": "software",
			"is_edited": false,
			"aliases": []
		}
	],
	"references": [
		"http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00035.html",
		"https://access.redhat.com/articles/3422521",
		"https://access.redhat.com/errata/RHSA-2018:1268",
		"https://access.redhat.com/errata/RHSA-2018:1269",
		"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1112",
		"https://review.gluster.org/#/c/19899/1..2",
		"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1112",
		"https://bugzilla.redhat.com/show_bug.cgi?id=1570891"
	],
	"weaknesses": [
		"CWE-287"
	]
}